### PR TITLE
Update 2000-01-05-page-automation.md

### PR DESCRIPTION
**Pull request by koksuanlim**
Update element id in useragent.js

### CODE REVIEW

File: _posts/documentation/learn/2000-01-05-page-automation.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Because PhantomJS can load and manipulate a web page, it is perfect to carry out
 
 Since the script is executed as if it is running on a web browser, standard [DOM scripting](http://en.wikipedia.org/wiki/DOM_scripting) and [CSS selectors](http://www.w3.org/TR/css3-selectors/) work just fine.
 
-The following `useragent.js` example demonstrates reading the `textContent` property of the element whose *id* is `myagent`:
+The following `useragent.js` example demonstrates reading the `textContent` property of the element whose *id* is `qua`:
```
Comment (rajendraarora16) [inline _posts/documentation/learn/2000-01-05-page-automation.md]: What is `qua` here? please explain.

**Comment by gidcs** [comment]
I am sorry for missing this issue. I just reply you.

File: _posts/documentation/learn/2000-01-05-page-automation.md (hunk 1 above)
Comment (pixiuPL) [reply _posts/documentation/learn/2000-01-05-page-automation.md]: @gidcs if so, whats the whole point pf having it named differently? It only substitues `user-agent`, and introduces mess. Iam against it.

Comment (gidcs) [reply _posts/documentation/learn/2000-01-05-page-automation.md]: The website changed the id which stored the actual element.
I just want to fix the broken code.
If you think the broken code is fine, it is okay.

**Pull request merged**